packages/features/bookings/Booker/Booker.tsx

```diff
@@ -241,6 +245,7 @@ const BookerComponent = ({
           key="logo"
           className={classNames(
             "mb-6 mt-auto pt-6 [&_img]:h-[15px]",
+            hasDarkBackground ? "dark" : "",
```

Comment:
Adding dark class here would cause logo image to invert automatically as it has `dark:invert` class